oak_minirootfs_linux_init/README.md

```diff
@@ -0,0 +1,62 @@
+<!-- Oak Logo Start -->
+<!-- An HTML element is intentionally used since GitHub recommends this approach to handle different images in dark/light modes. Ref: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to -->
+<!-- markdownlint-disable-next-line MD033 -->
+<h1><picture><source media="(prefers-color-scheme: dark)" srcset="../docs/oak-logo/svgs/oak-logo-negative.svg?sanitize=true"><source media="(prefers-color-scheme: light)" srcset="../docs/oak-logo/svgs/oak-logo.svg?sanitize=true"><img alt="Project Oak Logo" src="../docs/oak-logo/svgs/oak-logo.svg?sanitize=true"></picture></h1>
+<!-- Oak Logo End -->
+
+# Shell with Alpine's Mini Root Filesystem 
+
+This example shows how to boot into a shell from Oak's stage0 firmware binary. We can achieve this by simply packaging [Alpine Linux](https://www.alpinelinux.org)'s mini root filesystem as the [initial RAM disk](https://en.wikipedia.org/wiki/Initial_ramdisk) (initramfs) disk with an appropriate init script. The easiest way to execute the initial RAM disk is to use
```

Comment:
The format checks are failing. Running `xtask format` should fix it.

Reply:
done.